### PR TITLE
Documentation for Graceful Motion

### PR DESCRIPTION
**Pull request by ajtudela**
Documentation for [#4021](https://github.com/ros-planning/navigation2/pull/4021)

### CODE REVIEW

File: configuration/packages/configuring-graceful-motion-controller.rst [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  ============== =============================
+  Type           Default                      
+  -------------- -----------------------------
+  double         0.75 
+  ============== =============================
+
+  Description
+    The difference in the path orientation and the starting robot orientation to trigger a rotate in place, if ``initial_rotation`` is enabled.
+
+:initial_rotation:
```
Comment (SteveMacenski) [inline configuration/packages/configuring-graceful-motion-controller.rst:160]: Appears twice? Should be final rotation?

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: configuration/packages/configuring-graceful-motion-controller.rst (hunk 1 above)
Comment (ajtudela) [reply configuration/packages/configuring-graceful-motion-controller.rst]: Fixed now!

**Comment by SteveMacenski** [comment]
@ajtudela thanks for the contribution! If you had a video of this controller in action, that would be great for an announcement post to the community about your new controller plugin! Its worth spending a few minutes to make a nice demo video (then, that could be added to the config guide page as an example!)

**Comment by ajtudela** [comment]
I totally forgot! I made this [video](https://youtu.be/P-iKCo4h56M)

I don't know if it worth it.

**Comment by SteveMacenski** [comment]
That video is a bit slow and glitchy (lots of jumps), if you could make another one at more reasonable speeds and smoothness, that would be great!

**Comment by ajtudela** [comment]
Hi, I didn't forget about the video. It seems that YTdecrese the bitrate too much. So, I'd try to make a new video during this week.

**Comment by SteveMacenski** [comment]
It seems a bit wobbly when there are small deviations in the path that the control point jitters between. Is there a nice way to smooth that out? (@mikeferguson perhaps a nice challenge when you're working on the dynamic point setting)

It definitely seems like keeping the point further off is a good move.

**Pull request merged**